docs/Error codes.md

```diff
@@ -42,10 +42,10 @@ big sequence of items for each binding within a big tuple,
 which would lead to a data flow explosion and to a nesting of jobs on the Spark cluster.
 
 - [RBST0004] - Implementation Error.
-We should investigate this 🙈. Please contact us or file an issue on GitHub with your query.
-Link: https://github.com/RumbleDB/rumble/issues
 
-- [SENR0001] - Serialization error. Function items can not be serialized
+- [RBDY0005] - Materialization Error: the sequence is too big to be materialized. Use --result-size to increase the maximum  materialization size.
```

Comment:
If you agree as well, I was trying to keep an alphabetic order in Error codes.md and ErrorCodes.java files. Could you move this error before RBML errros?

Reply:
DONE